### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by maksym-nemera**
[DEMO LINK](https://maksym-nemera.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,21 @@
...
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      <p className="Person__partner">
+        {isMarried ? (
+          `${partnerName} is my ${spouse}`
+        ) : 'I am not married'
+        }
```
Comment (ruslanpashkov) [inline src/components/Person/Person.jsx:17]:
```suggestion
        {isMarried
          ? (
            `${partnerName} is my ${spouse}`
          ) : (
            'I am not married'
          )}
```

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const spouse = sex === 'm' ? 'wife' : 'husband';
```
Comment (ruslanpashkov) [inline src/components/Person/Person.jsx:5]:
```suggestion
  const {
    name,
    age,
    sex,
    isMarried,
    partnerName,
  } = person;

  const spouse = sex === 'm' ? 'wife' : 'husband';
```

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,21 @@
...
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      <p className="Person__partner">
```
Comment (ruslanpashkov) [inline src/components/Person/Person.jsx:13]: let's add some spaces between blocks

File: src/components/Person/Person.jsx [hunk 4]
```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const spouse = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
```
Comment (ruslanpashkov) [inline src/components/Person/Person.jsx:9]: let's move our jsx expression on new line for better readability
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
```

**Review by ruslanpashkov — CHANGES_REQUESTED** [review]

**Review by dinaminahi — CHANGES_REQUESTED** [review]
Great job! 👍 
@ruslanpashkov gave valid comments for better readability, let's fix them and we are good to go

**Review by dinaminahi — APPROVED** [review]
Great job 👍

**Pull request closed without merging**